string src
memory/TermMetadata meta
---
memory/TermMetadata[] metas
